src/celeritas/field/RZMapField.hh

```diff
@@ -53,18 +50,16 @@ class CMSMapField
  * Construct with the shared magnetic field map data (FieldMapData).
  */
 CELER_FUNCTION
-CMSMapField::CMSMapField(FieldMapRef const& shared) : shared_(shared) {}
+RZMapField::RZMapField(FieldMapRef const& shared) : shared_(shared) {}
 
 //---------------------------------------------------------------------------//
 /*!
- * Retrieve the magnetic field value for the given position.
+ * Retrieve the magnetic field value (in tesla) for the given position.
```

Comment:
The result should be "native" units, which I think is 0.001 T? The unit conversion on input may have been broken when I resolved the conflicts...

Reply:
I thought that the return value is now in tesla as we dropped cerlitas::units::tesla here assuming that input field values are in tesla unit, which am confused - of course, we need a proper conversion for the field value between Geant4 (T=0.001) and celeritas (tesla=1e+4?) for the CMS integration.

Reply:
We should take the reader input in Tesla and convert in the Params constructor